taxcalc/params.json

```diff
@@ -182,7 +182,7 @@
     },
      "_AMT_em_pe":{
         "long_name": "AMT exemption phaseout ending AMTI (Married filling Separately)",
-	    "description": "Individuals with married filing separately need to add extra amount to AMTI beyond this AMTI level",
+	"description": "Individuals with married filing separately need to add extra amount to AMTI beyond this AMTI level",
```

Comment:
Indeed exemption is entirely phased out beyond this AMTI level. In addition to that, they also need to add additional amount on their AMTI at the exemption phaseout rate for each extra dollar. This additional AMTI is capped at their exemption amount. Do we need to add this to the description? Or there's no need to make it that complicated here?